heartbeat/nagios4

```diff
@@ -0,0 +1,606 @@
+# OCF_ROOT=/usr/lib/ocf /usr/lib/ocf/resource.d/heartbeat/nagios
+# DESCRIPTION
+#       
+#
+# OPTIONS
+#
+#
+# RETURN CODE
+#     OCF_SUCCESS (0)
+#     OCF_ERR_GENERIC (1)
+#     OCF_ERR_ARGS (2)
+#     OCF_ERR_UNIMPLEMENTED (3)
+#     OCF_ERR_PERM (4)
+#     OCF_ERR_INSTALLED (5)
+#     OCF_ERR_CONFIGURED (6)
+#     OCF_NOT_RUNNING (7)
+#     OCF_RUNNING_MASTER (8)
+#     OCF_FAILED_MASTER (9)
+#
+#  REFERENCE:
+#   1. http://www.linux-ha.org/doc/dev-guides/ra-dev-guide.html
+#   2. TBC: link to an old OCF script for Nagios 3
+#
+# ---------------------------- CONSTANT DECLARATION ---------------------------
+# WHAT: Define PATH.
+# WHY:  Mainly for security reasons.
+# NOTE: This should only be changed if you know what you're doing.
+#
+
+# WHAT: Define exit status flags.
+# WHY:  Easier to program with a mnemonic than with the numbers.
+# NOTE: THESE DO NOT CHANGE UNLESS NOTIFIED BY HP!  Startup depends on these
+#       exit statuses being defined this way.
+
+: ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
+#[root@nagios1 heartbeat]# locate  .ocf-shellfuncs
+#/usr/lib/ocf/resource.d/heartbeat/.ocf-shellfuncs
+#[root@nagios1 heartbeat]#
+#
+. ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
+# ocf_is_root() {
+# ocf_maybe_random() {
+# ocf_is_decimal() {
+# ocf_is_true() {
+# ocf_is_hex() {
+# ocf_is_octal() {
+# ocf_log() {
+# ocf_exit_reason()
+# ocf_deprecated() {
+# ocf_run() {
+# ocf_pidfile_status() {
+# ocf_take_lock() {
+# ocf_release_lock_on_exit() {
+# ocf_is_probe() {
+# ocf_is_clone() {
+# ocf_is_ms() {
+# ocf_is_ver() {
+# ocf_ver2num() {
+# ocf_ver_level(){
+# ocf_ver_complete_level(){
+# ocf_version_cmp() {
+# ocf_local_nodename() {
+# ocf_check_level()
+# ocf_stop_processes() {
+# ocf_is_bash4() {
+# ocf_trace_redirect_to_file() {
+# ocf_trace_redirect_to_fd() {
+# ocf_default_trace_dest() {
+# ocf_start_trace() {
+# ocf_stop_trace() {
+# ocf_is_true "$OCF_TRACE_RA" && ocf_start_trace
+
+# Fill in some defaults if no values are specified
+# cutomized for CentOS 7.1, Nagios 3.5.1
+: ${OCF_RESKEY_installdir="/usr/bin"}
+: ${OCF_RESKEY_vardir="/var/nagios"}
+: ${OCF_RESKEY_binary="/usr/bin/nagios"}
+: ${OCF_RESKEY_cgidir="/usr/lib64/nagios/cgi-bin"}
+NAGIOS_BINDIR=`dirname ${OCF_RESKEY_binary}`
+: ${OCF_RESKEY_config="/etc/nagios/nagios.cfg"}
+: ${OCF_RESKEY_statusfile="/var/nagios/status.dat"}
+: ${OCF_RESKEY_statuslog="/var/nagios/livestatus.log"}
+: ${OCF_RESKEY_tempfile="/var/nagios/nagios.tmp*"}
+: ${OCF_RESKEY_retentionfile="/var/nagios/retention.dat"}
+: ${OCF_RESKEY_commandfile="/var/nagios/cmd/nagios.cmd"}
+: ${OCF_RESKEY_tempfile="/var/nagios/nagios.tmp"}
+: ${OCF_RESKEY_plugindir="/usr/lib64/nagios/plugins"}
+: ${OCF_RESKEY_user="nagios"}
+: ${OCF_RESKEY_group="nagios"}
+: ${OCF_RESKEY_PidFile="/var/nagios/nagios.pid"}
+
+OCF_RESKEY_PidFile="/var/nagios/nagios.pid"
+
+#
+# ---------------------------- VARIABLE DECLARATION ---------------------------
+#
+# WHAT: enable oracle tracing.
+#export ORACLE_TRACE='T'
+#
+SH=/bin/bash
```

Comment:
Do you need this variable? Also, does this script require bash? If so the top of this file should have !/bin/bash instead of !/bin/sh